source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+.. _install-kafka-cluster:
+
+======================================
+Install with an Existing Kafka Cluster
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on an existing Kafka cluster including 
+Kafka Connect. This installation method is ideal if you are already running 
+Kafka or you have your own approach to installing and configuring Kafka.
+
+Before you Begin 
+----------------
+
+You must have Kafka and a Kafka Connect cluster accessible to the computer 
+running Relational Migrator.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install Debezium Source Connect Plugin
+
+      1. Download the source connector archive for your source database.
+
+         - `MySQL Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/2.4.1.Final/debezium-connector-mysql-2.4.1.Final-plugin.tar.gz>`__
+         - `Postgres Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/2.4.1.Final/debezium-connector-postgres-2.4.1.Final-plugin.tar.gz>`__
+         - `SQL Server Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/2.4.1.Final/debezium-connector-sqlserver-2.4.1.Final-plugin.tar.gz>`__ 
+         - `Oracle Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-oracle/2.4.1.Final/debezium-connector-oracle-2.4.1.Final-plugin.tar.gz>`__  
+         - `Db2 Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-db2/2.4.1.Final/debezium-connector-db2-2.4.1.Final-plugin.tar.gz>`__
```

Comment:
**Suggestion**

I know DB2 is in scope for support in the near future, but we currently dont cover or list them in the [supported database](https://www.mongodb.com/docs/relational-migrator/getting-started/supported-databases/). If we leave this in, can we file a fast follow ticket to add DB2 to supported database / connection string pages?

Reply:
Yes thank you, it has been created: https://jira.mongodb.org/browse/DOCSP-37778 :)